docs/book/how-to/manage-the-zenml-server/using-zenml-server-in-prod.md

```diff
@@ -0,0 +1,191 @@
+---
+description: >
+  Learn about best practices for using ZenML server in production environments.
+---
+
+# Using ZenML server in production
+
+Setting up a ZenML server is pretty straightforward. However, most people have to move ahead of day zero operations and in such cases, it helps to learn best practices around setting up your ZenML server in a production-ready way. This guide encapsulates all the tips and tricks we've learned ourselves and from working with people who use ZenML in production environments. Following are some of the best practices we recommend.
```

Comment:
Dont say "its striaghtforward" and then have a huge guide :-) no need to take emotional opinions I think

Reply:
ah, i agree. i wanted it to mean that just deploying vanilla zenml server is easy but then managing and maintaining it is hard. let me rephrase that